pycbc/workflow/minifollowups.py

```diff
@@ -560,8 +560,6 @@ def make_sngl_ifo(workflow, sngl_file, bank_file, trigger_id, out_dir, ifo,
     node.add_input_opt('--single-trigger-file', sngl_file)
     node.add_input_opt('--bank-file', bank_file)
     node.add_opt('--trigger-id', str(trigger_id))
-    if rank is not None:
```

Comment:
surely if you remove the rank=None default value the code will say "what's this object called 'rank'" here?

Reply:
This line is also being removed (did I miss something??)

Reply:
oh, duh. so, no more rank for anything.